content/docs/guides/continuous-delivery/teamcity.md

```diff
@@ -14,14 +14,14 @@ This page details how to use [JetBrains TeamCity](https://www.jetbrains.com/team
 
 - A working installation of TeamCity
 - An account on the [Pulumi Console](https://app.pulumi.com).
-- The latest version of Pulumi. Installation instructions are [here]({{< relref "/docs/get-started/install" >}}).
+- [Latest version of Pulumi]({{< relref "/docs/get-started/install" >}}).
 - Setup a new project and [stack]({{< relref "/docs/intro/concepts/stack" >}}) using one of our
 [Get Started]({{< relref "/docs/get-started" >}}) guides or by running [`pulumi new`]({{< relref "/docs/reference/cli/pulumi_new.md" >}})
 and choosing one of the many templates that are available.
 
 ## Sample Project
 
-The example we are going to deploy is located [here](https://github.com/pulumi/examples/tree/master/aws-ts-hello-fargate).
+[Example Repository](https://github.com/pulumi/examples/tree/master/aws-ts-hello-fargate).
```

Comment:
Yeah, I think leading with a complete sentence is a good thing in general. Explanatory fragments like are totally fine  tacked onto the ends of paragraphs and such -- e.g., "[...] More about projects." -- but up front, they don't work in quite the same way. At least for me.